bedrock/firefox/templates/firefox/switch.html

```diff
@@ -16,61 +18,70 @@
+    </ol>
+    <div class="c-steps-cta">
+    {{ download_firefox(alt_copy=_('Download and switch'), download_location='primary cta') }}
+    </div>
+  </section>
 
-        {{ download_firefox(alt_copy=_('Download and switch'), download_location='primary cta') }}
+  <section class="c-share">
+    <div class="mzp-l-content">
 
-        <footer>
-          <p>{{ _('You can use Firefox and still have Chrome. Chrome won’t change on your machine one bit.') }} {{ _('Share with your friends how to switch to Firefox') }}</p>
+      {% if l10n_has_tag('switch_201908') %}
+        <h2 class="c-share-title">{{ _('Share your Firefox experience.') }}</h2>
+        <p>{{ _('Tell your friends and family about the easy switch.') }}</p>
+      {% else %}
+        <p>{{ _('You can use Firefox and still have Chrome. Chrome won’t change on your machine one bit.') }} {{ _('Share with your friends how to switch to Firefox') }}</p>
+      {% endif %}
 
-          {% set share_url = settings.CANONICAL_URL + canonical_path %}
-          <ul class="share-options">
-            <li>
-              <a class="facebook" data-link-type="social share" data-link-name="Facebook" href="https://www.facebook.com/sharer/sharer.php?s=100&p[url]={{ share_url|urlencode }}">
-                {{ _('Share to Facebook') }}
-              </a>
-            </li>
-            <li>
-              {# L10n: the string below is for a referral Twitter message. #}
-              {% set tweet_text = _('🔥 Firefox makes switching from Chrome really fast. Try it out!') %}
-              <a class="twitter" data-link-type="social share" data-link-name="Twitter" href="https://www.twitter.com/intent/tweet?url={{ share_url|urlencode }}&text={{ tweet_text|urlencode }}">
-                {{ _('Send a tweet') }}
-              </a>
-            </li>
-            <li>
-              {# L10n: the string below is the subject line for an email referral message. #}
-              {% set email_subject = _('Switch to Firefox')|mailtoencode %}
-              {# L10n: the string below is the introduction of the email referral message body. #}
-              {% set email_intro = _('Hey,')|mailtoencode %}
-              {# L10n: the string below is the first sentance of the email referral message body. #}
-              {% set email_start = _('Firefox makes switching from Chrome really fast. I like it a lot, and you should try it.')|mailtoencode %}
-              {# L10n: the string below is the second sentance of the email referral message body. At the end of the string after the colon, a URL will be added automatically. #}
-              {% set email_end = _('Check it out and let me know what you think:')|mailtoencode %}
+      {% set share_url = settings.CANONICAL_URL + canonical_path %}
+      <ul class="c-share-options">
+        <li>
+          <a class="facebook" data-link-type="social share" data-link-name="Facebook" href="https://www.facebook.com/sharer/sharer.php?s=100&p[url]={{ share_url|urlencode }}">
```

Comment:
My Facebook Container extension blocks this link:

![image](https://user-images.githubusercontent.com/400117/64021779-9fb5c480-cb2c-11e9-8f34-a24806ad6049.png)

Considering we no longer promote on Facebook any longer or advertise there, should we consider removing this link?